packages/cerebral/package.json

```diff
@@ -35,5 +35,8 @@
       "**/*.test.js",
       "**/testHelper.js"
     ]
+  },
+  "devDependencies": {
+    "prop-types": "^15.5.10"
```

Comment:
You need to add
```json 
    "prop-types": "15.5.10"
```
also to devDependencies in root package.json. This time without caret (`^`). 
Otherwise dependency test fails. 

For details ask @gaspard.